react/components/ShippingSimulator/index.js

```diff
@@ -88,25 +90,27 @@ const ShippingSimulator = ({
             <PostalCodeGetter />
           </AddressContainer>
         </AddressRules>
-        <Button
-          onClick={handleClick}
-          className={styles.shippingCTA}
-          disabled={!isValid}
-          size="small"
-          type="submit"
-          block
-          isLoading={loading}
-        >
-          {intl.formatMessage({ id: 'store/shipping.label' })}
-        </Button>
+        <div className={handles.shippingButtonContainer}>
```

Comment:
This additional div can cause some problems

Reply:
vi nas lojas e não vai afetar ngm